syntax = "proto3";
//python -m grpc_tools.protoc -I./ --python_out=. --grpc_python_out=. ./InsultAPI.proto
message InsultListMessage {
    repeated string list = 1;
}

message RandInsultMessage {
    string message = 1;
}

message Request {
    string requestName = 1;
}

service InsultAPI {
    rpc getInsults(Request) returns (InsultListMessage) {}
    rpc insultme(Request) returns (RandInsultMessage) {}
    rpc addInsult(Request) returns (RandInsultMessage) {}
}